app/graphql/types/failed_create_rating_question_result.rb

```diff
@@ -0,0 +1,9 @@
+module Types
+  class FailedCreateRatingQuestionResult < BaseObject
+    field :error, String, null: true
+  end
+
+  def errors
+    object.errors.full_message
```

Comment:
This type looks the same as `Types::ErrorMessage`. Perhaps you could converge on using that one type now rather than creating many different type classes that do roughly the same thing?